features/admin.core.v1/components/header.tsx

```diff
@@ -150,7 +177,11 @@ export const Header: FunctionComponent<HeaderPropsInterface> = (props: HeaderPro
         (organizationType === OrganizationType.SUPER_ORGANIZATION ||
             organizationType === OrganizationType.FIRST_LEVEL_ORGANIZATION ||
             organizationType === OrganizationType.SUBORGANIZATION ||
-            organizationConfigs.showSwitcherInTenants) && hasOrganizationReadPermission
+            organizationConfigs.showSwitcherInTenants) &&
+        hasOrganizationReadPermission
```

Comment:
Need to enable the org switcher for everyone in the primary userstore with console access

Reply:
Fixed with [3b2069d](https://github.com/wso2/identity-apps/pull/7141/commits/3b2069d00f35b0b67efa66c0c54556103aaac65f)